.github/workflows/main.yml

```diff
@@ -31,34 +31,44 @@ jobs:
         echo Using Ktlint $KTLINT
         curl -sSLO https://github.com/pinterest/ktlint/releases/download/$KTLINT/ktlint
         chmod a+x ktlint
-    
-    -  name: Download buf
-       run: |
+
+    - name: Download buf
+      run: |
         BUFVERSION="v0.15.0"
         echo Using Bufbuild version $BUFVERSION
         curl -sSL https://github.com/bufbuild/buf/releases/download/$BUFVERSION/buf-$(uname -s)-$(uname -m) > buf
         chmod a+x buf
-        
+
+    - name: Download Checkstyle
+      run: |
+        CHECKSTYLE="8.37"
+        echo Using Checkstyle version $CHECKSTYLE
+        curl -sSLO https://github.com/checkstyle/checkstyle/releases/download/checkstyle-$CHECKSTYLE/checkstyle-$CHECKSTYLE-all.jar > checkstyle
+        chmod a+x checkstyle
+
     - name: Kotlin lint check
       run: ./ktlint --android domain/src/**/*.kt utility/src/**/*.kt data/src/**/*.kt app/src/**/*.kt testing/src/**/*.kt && echo "Lint completed successfully"
-    
-    -  name: Protobuf lint check
-       run: ./buf check lint --input=model/src/main/proto --input-config buf.yaml && echo "Protobuf lint check completed successfully"
-     
+
+    - name: Protobuf lint check
+      run: ./buf check lint --input=model/src/main/proto --input-config buf.yaml && echo "Protobuf lint check completed successfully"
+
+    - name: Java lint check
+      run: ./checkstyle -c=checkstyle.xml app/src/main/java/org/oppia/android/app/databinding/DrawableBindingAdapters.java && echo "Lint completed successfully"
```

Comment:
command should be 
```console
foo@bar:~$ 
java -jar checkstyle-8.37-all.jar -c=checkstyle.xml 
app/src/main/java/org/oppia/android/app/databinding/DrawableBindingAdapters.java
```

Reply:
Done, looks like it is working now, but it should fail on these warning!! Checking into this

<img width="1078" alt="Screenshot 2020-11-10 at 16 03 23" src="https://user-images.githubusercontent.com/25057618/98662848-5edf4f80-236e-11eb-8821-82028a8edf0b.png">

Reply:
on local machine, 

```console
java --version
java 10.0.2 2018-07-17
Java(TM) SE Runtime Environment 18.3 (build 10.0.2+13)
Java HotSpot(TM) 64-Bit Server VM 18.3 (build 10.0.2+13, mixed mode)
```

```console
Starting audit...
[WARN] DrawableBindingAdapters.java:22:3: 'method def rcurly' has incorrect indentation level 2, expected level should be 4. [Indentation]
[WARN] DrawableBindingAdapters.java:24:3: Missing a Javadoc comment. [MissingJavadocMethod]
[WARN] DrawableBindingAdapters.java:30:3: 'method def rcurly' has incorrect indentation level 2, expected level should be 4. [Indentation]
[WARN] DrawableBindingAdapters.java:32:3: Missing a Javadoc comment. [MissingJavadocMethod]
[WARN] DrawableBindingAdapters.java:38:3: 'method def rcurly' has incorrect indentation level 2, expected level should be 4. [Indentation]
[WARN] DrawableBindingAdapters.java:40:3: Missing a Javadoc comment. [MissingJavadocMethod]
[WARN] DrawableBindingAdapters.java:46:3: 'method def rcurly' has incorrect indentation level 2, expected level should be 4. [Indentation]
[WARN] DrawableBindingAdapters.java:51:3: 'method def rcurly' has incorrect indentation level 2, expected level should be 4. [Indentation]
[WARN] DrawableBindingAdapters.java:52:1: 'class def rcurly' has incorrect indentation level 0, expected level should be 2. [Indentation]
Audit done.
```

Reply:
updating java version not worked

Reply:
From our `Project.xml`, the indent is 2 but warning required 4 as per google java style.

```xml
<codeStyleSettings language="JAVA">
      <indentOptions>
        <option name="INDENT_SIZE" value="2" />
        <option name="CONTINUATION_INDENT_SIZE" value="4" />
        <option name="TAB_SIZE" value="2" />
      </indentOptions>
    </codeStyleSettings>
```

Is there any way to update project.xml and reflect the same in all files?

Reply:
<img width="1037" alt="Screenshot 2020-11-10 at 16 40 04" src="https://user-images.githubusercontent.com/25057618/98666731-70772600-2373-11eb-898b-9c2343d54866.png">

Reply:
the pattern is, if the warning is something related to **rcurly** , it's not coming in checks
1. 'method def rcurly'
2. 'class def rcurly'

Reply:
<img width="1035" alt="Screenshot 2020-11-10 at 17 29 26" src="https://user-images.githubusercontent.com/25057618/98671465-51c85d80-237a-11eb-96a4-7456108eaddd.png">